tailwind.config.js

```diff
@@ -173,6 +173,22 @@ module.exports = {
       opacity: {
         '10': '0.1',
         '60': '0.6'
+      },
+      keyframes: {
+        appear: {
+          '0%': {
+            'opacity': '0',
+            'pointer-events': 'none'
+          },
+          '100%': {
+            'opacity': '1',
+            'pointer-events': 'auto'
+          }
```

Comment:
When the key is a single word, with no special characters, we don't need to add it as a string (note the `opacity` key right above, on line 173)
```suggestion
        appear: {
          '0%': {
            opacity: '0',
            'pointer-events': 'none'
          },
          '100%': {
            opacity: '1',
            'pointer-events': 'auto'
          }
```